<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::SkillChannel</class>
 <widget class="QWidget" name="ns::SkillChannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>53</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetMain {
	background: #E6F3FC; 
}

#labelMovie {
	border-image: url(:/images/ui/waiting.gif);
}

#labelSource, #labelTarget, #labelLocalTarget {
	font-weight: bold;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>53</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>53</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetBottom" native="true">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>53</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>53</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QWidget" name="widgetLanguage" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelLocalTarget">
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSource">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSplitter">
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>10</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTarget">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetState" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelMovie">
              <property name="minimumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelState">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Waiting</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNoService">
           <property name="text">
            <string>Empty</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
